### PR TITLE
feat: added github link to header #1632

### PR DESCRIPTION
**Pull request by iamayushdas**
issue resolved from #1632 

Here are some minor issues on the Login page we could resolve.

## TODO
 * [ ]  Left icon replacement. @LiteSun
 * [ ]  Extracting Table as a separate component for reuse in various modules.
 * [ ]  APISIX plugin type change.
 * [x]  GitHub hyperlink (icon) added to Header section.

### CODE REVIEW

**Comment by iamayushdas** [comment]
@juzhiyuan please review this

File: web/src/pages/User/Login.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ import logo from '@/assets/logo.svg';
 import { getUrlQuery } from '@/helpers';
 import Footer from '@/components/Footer';
 import styles from './Login.less';
+import { GithubOutlined } from '@ant-design/icons';
```
Comment (juzhiyuan) [inline web/src/pages/User/Login.tsx:28]: Hi, would better move this line to the top :)

Comment (iamayushdas) [reply web/src/pages/User/Login.tsx:28]: Done!

**Comment by LiteSun** [comment]
It would be great if you could post a screenshot of the page 😁

**Pull request closed without merging**